config.js

```diff
@@ -648,9 +649,6 @@ var config = {
     // Enables sending participants' emails (if available) to callstats and other analytics
     // enableEmailInStats: false,
 
-    // When true, disables the connection indicator popover.
-    // disableConnectionIndicatorDetails: false,
```

Comment:
Had you whitelisted it before? If so I guess you need to remove it.

Reply:
No, it was not whitelisted.